client/src/canvas/View.res

```diff
@@ -634,33 +647,56 @@ let accountView = (m: model): Html.html<msg> => {
 
   let tutorial = Html.p(
     list{
-      Attrs.class("account-action-btn"),
+      tw(accountActionBtnStyle),
       EventListeners.eventNoPropagation(~key="tutorial", "click", _ => Msg.ToolTipMsg(
         UpdateTutorial(ReopenTutorial),
       )),
     },
     list{Html.text("Hello World tutorial")},
   )
 
-  let spacer = Html.div(list{Attrs.class("account-action-spacer")}, list{})
+  let spacer = Html.div(list{tw(%twc("bg-grey2 h-px my-2.5 mx-0"))}, list{})
   let settings = Html.p(
     list{
-      Attrs.class("account-action-btn"),
+      tw(accountActionBtnStyle),
       EventListeners.eventNoPropagation(~key="open-settings", "click", _ => Msg.SettingsMsg(
         Open(Canvases),
       )),
     },
     list{Html.text("Settings")},
   )
 
-  let share = Html.p(
+  let reportBugRef = Html.a(
+    list{
+      tw(accountActionBtnStyle),
+      Attrs.href("https://github.com/darklang/dark/issues"),
+      Attrs.target("_blank"),
+      EventListeners.eventNoPropagation(~key="github-issues-ref", "click", _ => Msg.UpdateHeapio(
+        OpenKeyboardRef,
```

Comment:
The OpenKeyboardRef listener isn't the right one - but you can just remove this an use no listener.

Reply:
What would be the right listener if we wanted to have one?

Reply:
You'd add a new value to the place where `OpenKeyboardRef` is defined

Reply:
Ok, Thanks.